torchinductor/lowering.py

```diff
@@ -125,7 +125,9 @@ def decode_device(device):
     return device
 
 
-def get_promoted_dtype(*args):
+def get_promoted_dtype(
+    *args, type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.DEFAULT
```

Comment:
don't default type promotion type here, let's be explicit (in the only callsite it's defined anyway).

Reply:
fixed.